css/style.css

```diff
@@ -0,0 +1,85 @@
+
+* {
+  font-family: 'Inter', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.mob-navbar {
+  display: none;
+}
+
+@media (max-width: 768px) {
+  nav img {
+    width: 18px;
+    height: 12px;
```

Comment:
- [ ] [REQUIRED] Kindly, remove the `@media` query and put the code with no `@media`. Since in future milestones of the project, you will be required to add media quaries for desktop version only. This is called mobile-first design. You can check more about it [here](https://css-tricks.com/how-to-develop-and-test-a-mobile-first-design-in-2021/#:~:text=What%20is%20a%20Mobile-First%20Design%3F%20Mobile-first%20design%20is,surge%20in%20mobile%20internet%20users%20around%20the%20world.)